lib/models/record.js

```diff
@@ -77,6 +78,13 @@ class Record extends BaseModel {
       (r) => new Server({ id: r.record_info.server_id })
     );
   }
+
+  splits() {
+    if (this.attrs.splits) {
```

Comment:
If this isn't available and `rank` is 1, we should use the resource again to fill this in. It might be a little tricky because it can come from `this.access(...)` or `this.attrs` depending on the context, but I don't think it's too bad.

I have been meaning to rethink how some of the record loading/caching works because the various ways that things can be preloaded makes maintaining this all a pain.